docs/playground-old.md

```diff
@@ -0,0 +1,270 @@
+
+#### The rule does not behave as expected or does not find the desired match
+
+File a [bug](https://github.com/returntocorp/semgrep/issues/new?title=semgrep.dev%20bug%20report) or reach out through [Semgrep Community Slack](https://r2c.dev/slack).
+
+
+### Evaluating a rule's performance
+
+After running Semgrep on the test code, a summary of findings appears under the **Matches** column.
+
+The **Performance **column displays how long it takes in seconds for Semgrep to run your rule. It also displays the following benchmarks:
+
+* Blazing
+* Quick
+* Decent
+* Fair
+* Okay
+* Slow
+* Sluggish
+
+These benchmarks take into account the **visual complexity** of the rule.
+
+
+## Setting code standards by adding a rule to the Rule Board
+
+The [Rule Board](../semgrep-app/rule-board/) displays rules that Semgrep App uses to scan your project's code. Adding a rule to the Rule Board allows you to quickly set a rule as part of every Semgrep scan. A rule on the Rule Board is represented as a card and can be placed on either the Audit Board or the Block board depending on the importance of the rule. Rules can be dragged from one column to another.
+
+To add a rule to the Rule Board:
+
+1. Ensure that you are logged-in.
+2. Click **Add to Rule Board**.
+3. Select either the Audit or Block boards depending on the importance of the rule. Matches for rules on the Block board will prevent PR merges within a CI environment.
+
+## Saving and sharing a Semgrep rule
+
+A rule is saved by either **explicitly naming it** or when it is **assigned a shortcode** for sharing. Rules are **unlisted** by default. This means they do not appear publicly in the Registry nor can they be searched for, but are accessible through their sharing URL.
+
+To explicitly name a rule:
+
+1. Sign into Semgrep.
+2. Click the **Untitled rule** box.
+3. Enter the name of your rule and click **Save**.
+
+To assign a random shortcode:
+
+1. Click **Share**. This auto-generates a random shortcode and URL to access the rule.
+
+To retrieve or open a rule:
+
+1. Click **File > Open**
+2. Enter the rule's name in the **Filter** search box or scroll to find the rule.
+3. Click on the rule. The rule loads into the Playground.
+
+To rename a rule:
+
+1. Sign into Semgrep.
+2. In the Playground, open the rule to rename.
+3. Enter a new name and press **Save**. This creates a new rule.
+4. Delete the old rule to complete the renaming process.
+
+
+## Using a rule in CI
+
+Using a rule in your CI pipeline enables you to **run the rule at a certain hook or event**. This allows users to add guardrails against security, style, and general code violations.
+
+Semgrep can be used in the following CI environments:
+
+* GitHub (pre-commit and workflows)
+* GitLab
+* CircleCI
+* AppVeyor
+* TravisCI
+
+### Prerequisites
+
+
+1. Ensure that you have sufficient permissions to update your CI environment.
+
```

Comment:
Take the commit suggestion just as an idea - making this ad hoc header here - so this will not be only one thing with the Prerequisites (you can use a simple Procedure header if nothing else will fit here, but including the actions to take under Prerequisites header is not good). @s-santillan